internal/appsec/dyngo/instrumentation/httpinstr/http.go

```diff
@@ -60,50 +60,17 @@ func init() {
 // WrapHandler wraps the given HTTP handler with the abstract HTTP operation defined by HandlerOperationArgs and
 // HandlerOperationRes.
 func WrapHandler(handler http.Handler, span ddtrace.Span) http.Handler {
+	// TODO: move these to service entry tags
```

Comment:
Can you please leave your Github username? Check point 5 in https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#general-guidelines

Reply:
Fixed